samples/azure-quantum/check-ghz/CheckGHZ.ipynb

```diff
@@ -0,0 +1,191 @@
+{
+  "cells": [
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "# Integrated Hybrid: Blending classical and quantum computing\n",
+        "In this sample, you will discover how to blend classical and quantum instructions in the same program, all fully processed by the quantum computing back-end.\n",
+        "You can learn more how to program with Integrated hybrid at https://aka.ms/AQ/Hybrid/Docs\n",
+        "\n",
+        "This program will examine a 3 qubit Greenberger–Horne–Zeilinger (GHZ) state and counts the number of times it sees the entanglement fail out of 10 attempts. Unlike in perfect, noiseless simulations, entanglement failures can occur in real-world runs, which are subject to the effects of noise.\n",
+        "\n",
+        "In the CheckGHZ() operation, you see three capabilities of Integrated Hybrid being demonstrated:\n",
+        "1. **Branching**: based on measurement results you can adjust the classical code being performed\n",
+        "```Q#\n",
+        "   if not (r0 == r1 and r1 == r2)\n",
+        "```\n",
+        "2. **Classical comput**e: here an arithmetic operation is performed\n",
+        "```Q#\n",
+        "   set mismatch += 1;\n",
+        "```\n",
+        "3. **While the physical qubits are still coherent**, #1 and #2 are performed \n",
```

Comment:
To the point @anpaz brought up in a different review, this is technically not true, since the qubits are reset during the measurement. I think there was suggestion to use a different sample here, but I'm not sure if that has been decided there already. For now, I think this phrasing could be distracting to folks who would take issue with it.

Reply:
I think it was decided to include this sample, so I changed the description to make it accurate and representative of what the program is actually doing.